include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -14,6 +14,10 @@
 //===----------------------------------------------------------------------===//
 
 //!!! NOTE: This file should be included under the macro _ONEDPL_BACKEND_SYCL
+
+// The definition of _ONEDPL_parallel_backend_sycl_H means the inclusion of this
```

Comment:
The comment seems to long to me. Furthermore, changing that macro doesn't change the behavior of the product.

Consider changing to something like (the example of FPGA backend):
```cpp
// This header guard is used to check inclusion of DPC++ backend for the FPGA.
// Changing this macro may result in broken tests.
```

There is no need to mention SYCL, we call heterogeneous backend DPC++ one.

This review note is relevant to all headers guards used in the test

Reply:
fix in last commit